helm/default-apps-cloud-director/values.yaml

```diff
@@ -3,17 +3,38 @@ organization: ""
 managementCluster: ""
 
 userConfig:
+  certExporter:
+    configMap:
+      values: |
+        ciliumNetworkPolicy:
+          enabled: true
+  certManager:
+    configMap:
+      values: |
+        ciliumNetworkPolicy:
+          enabled: true
+  metricsServer:
+    configMap:
+      values: |
+        ciliumNetworkPolicy:
+          enabled: true
+  netExporter:
+    configMap:
+      values: |
+        ciliumNetworkPolicy:
+          enabled: true
+        dns:
+          service: kube-dns
   nodeExporter:
     configMap:
       values: |
         disableConntrackCollector: true
         disableNvmeCollector: true
-  certManager:
+  vpa:
     configMap:
       values: |
-        controller:
-          defaultIssuer:
-            name: letsencrypt-giantswarm
```

Comment:
don't we need the default issuer here?

Reply:
https://github.com/giantswarm/cert-manager-app/blob/8adabf2af43ce3c6c19f94d0bba5e4064fc90a45/helm/cert-manager-app/values.yaml#L73